src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerCreateUserWithFileUploadTest.java

```diff
@@ -146,6 +146,12 @@ public void shouldReturn400WhenFileHasNoData() throws IOException {
             TYPE_XLSX, "400", cwdAdmin);
     }
 
+    @Test
+    public void shouldReturn400WhenFileHasEmptyRows() throws IOException {
+        uploadCaseWorkerFile("Staff Data Upload With Empty Rows.xlsx",
+            TYPE_XLSX, "400", cwdAdmin);
```

Comment:
@ishratt These file contains multiple blank row right? also do we have some bank non blank excel integration test?

Reply:
Yes, this file contains only a header followed by empty rows. We currently don't have any integration tests covering blank rows in the middle as we can have only one valid entry per excel sheet, but I have done some manual tests in my local to check if the functionality is working as expected.

Reply:
Please add some integration test case. 1. all blanks which is here **2. Mixed BLANK And non Blank and we should Validated JSR errors**

Reply:
Sure, will do, thanks.